### PR TITLE
Update upgrading cookbook

### PR DESCRIPTION
**Pull request by azf20**
Update to recommend using the upgrade flow, including video

### CODE REVIEW

**Review by MichaelMacaulay — APPROVED** [review]

File: website/pages/en/cookbook/upgrading-a-subgraph.mdx [hunk 1]
```diff
@@ -10,98 +10,39 @@ The process of upgrading is quick and your subgraphs will forever benefit from t
...
 
-3. Generate files and build the subgraph:
+1. Select the subgraph(s) you want to upgrade.
+2. Connect or enter the receiving wallet (the wallet that will become the owner of the subgraph).
+3. Click the "Upgrade" button.
 
-```sh
-graph codegen && graph build
-```
+That's it! Your subgraphs will be deployed to the Subgraph Studio, and published on The Graph Network. You can access the [Subgraph Studio](thegraph.com/studio) to manage your subgraphs, logging in with the wallet specified during the upgrade process.
```
Comment (benface) [inline website/pages/en/cookbook/upgrading-a-subgraph.mdx:27]: Nitpick, but is it really "the" Subgraph Studio or just Subgraph Studio? cc @bermchain

File: website/pages/en/cookbook/upgrading-a-subgraph.mdx [hunk 2]
```diff
@@ -10,98 +10,39 @@ The process of upgrading is quick and your subgraphs will forever benefit from t
...
 
-If your subgraph has build errors, refer to the [AssemblyScript Migration Guide](/release-notes/assemblyscript-migration-guide/).
+You'll be able to view your subgraphs live on the decentralized network via [The Graph Explorer](https://thegraph.com/explorer).
```
Comment (benface) [inline website/pages/en/cookbook/upgrading-a-subgraph.mdx:29]: Pretty sure it is just `Graph Explorer` though. :) cc @bermchain 

```suggestion
You'll be able to view your subgraphs live on the decentralized network via [Graph Explorer](https://thegraph.com/explorer).
```

File: website/pages/en/cookbook/upgrading-a-subgraph.mdx [hunk 3]
```diff
@@ -10,98 +10,39 @@ The process of upgrading is quick and your subgraphs will forever benefit from t
...
-
-![Add GRT in billing](/img/Add-GRT-New-Page.png)
-
-- Follow the steps to add your GRT to your billing balance.
-- Your GRT will be automatically bridged to the Arbitrum network and added to your billing balance.
-
-![Billing pane](/img/New-Billing-Pane.png)
-
-> Note: see the [official billing page](../billing.mdx) for full instructions on adding GRT to your billing balance.
+> Note: see the [billing documentation](../billing.mdx) for more information on plans, and on managing your billing on the Subgraph Studio.
```
Comment (benface) [inline website/pages/en/cookbook/upgrading-a-subgraph.mdx:45]:
```suggestion
> Note: see the [billing documentation](../billing) for more information on plans, and on managing your billing on the Subgraph Studio.
```

Comment (benface) [reply website/pages/en/cookbook/upgrading-a-subgraph.mdx:45]: Or, probably even better:
```suggestion
> Note: see the [billing documentation](/billing) for more information on plans, and on managing your billing on the Subgraph Studio.
```

File: website/pages/en/sunrise.mdx [hunk 4]
```diff
@@ -12,7 +12,7 @@ This plan draws on many previous developments from The Graph ecosystem, includin
 
 ### What are the phases of the sunrise of decentralized data?
 
-**Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on The Graph Network.  
+**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:15]: Nitpick but the last item in the enumeration is missing a verb so what do you think about:

```suggestion
**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network as well as simple payment options.  
```

**Review by schmidsi — APPROVED** [review]

**Pull request merged**